libs/core/src/lib/form/form-label/form-label.component.ts

```diff
@@ -55,7 +55,7 @@ export class FormLabelComponent implements OnChanges {
     /** The trigger events that will open/close the inline help.
      *  Accepts any [HTML DOM Events](https://www.w3schools.com/jsref/dom_obj_event.asp). */
     @Input()
-    inlineHelpTrigger: string[] = ['mouseleave', 'mouseenter'];
+    inlineHelpTrigger: string[] = ['mouseleave', 'mouseenter', 'focusin', 'focusout'];
```

Comment:
I saw the same code in different place as well, if we have to add it to more and more places, may be it's better to extract it somewhere as a general setting and use everywhere? 

https://github.com/SAP/fundamental-ngx/pull/5863/files#diff-476fa704b9e826c820a27653c17d7ae879415a32f504c7e8e603ee063722400aR21

Reply:
![Screenshot 2021-07-08 195116](https://user-images.githubusercontent.com/53093915/124962056-74e90100-e01e-11eb-8214-f8a8f541b5d0.png)
@aberikashvili 
I can add 'focusin', 'focusout' into inline-help.directive. Please advise if it is good idea.